charts/incubator/channels-dvr/values.yaml

```diff
@@ -0,0 +1,27 @@
+image:
+  repository: fancybits/channels-dvr
+  pullPolicy: Always
+  tag: latest@sha256:5cad00679feb37533f0eccf72b70c3bf7acc4b4c18fcae65e2081e7222da39ce
+
+securityContext:
+  readOnlyRootFilesystem: false
+  runAsNonRoot: false
+
+podSecurityContext:
+  runAsUser: 0
+  runAsGroup: 0
```

Comment:
```suggestion
#securityContext:
#  readOnlyRootFilesystem: false
#  runAsNonRoot: false

#podSecurityContext:
#  runAsUser: 0
#  runAsGroup: 0
```
Does it require root?
If you are unsure, comment them out and see what CI will say

Reply:
Based on my testing using custom-app, I don't think it needs to run as root. I'm looking into the root file system.

Reply:
It does not need `readOnlyRootFilesystem` to be `false`. Change made.

Reply:
Which file system does this make read only? Software updates may require writes.

Reply:
@tmm1 The root (`/`) file system excluding `/channels-dvr` and excluding any mounts created by the user (for example a mount for storing the recordings). Will the update mechanism modify anything outside of `/channels-dvr`?

Reply:
Okay great. It will only modify things inside channels-dvr